RecoHI/HiJetAlgos/plugins/HiFJRhoFlowModulationProducer.cc

```diff
@@ -36,9 +38,51 @@
 #include <vector>
 
 namespace {
-  double lineFunction(double* x, double* par) { return par[0]; }
+
   double flowFunction(double* x, double* par) {
-    return par[0] * (1. + 2. * (par[1] * std::cos(2. * (x[0] - par[2])) + par[3] * std::cos(3. * (x[0] - par[4]))));
+    int nFlow = par[0];             // Number of fitted flow components is defined in the first parameter
```

Comment:
This sounds like it should always be positive.  
In that case use an unsigned int would be appropriate

Reply:
This is correct. uint can be used here.